swagger/body-formdata-urlencoded.json

```diff
@@ -53,5 +59,33 @@
         }
       }
     }
+  },
+  "parameters": {
+    "Pet_type": {
+        "name": "pet_type",
+        "description": "Can take a value of dog, or cat, or fish",
+        "type": "string",
+        "in": "formData",
+        "required": true,
+        "enum": [
+            "dog",
+            "cat",
+            "fish"
+        ],
+        "x-ms-parameter-location": "method"

```

Comment:
let's make one of these enums extensible to get a bit more coverage.

Reply:
see https://github.com/stankovski/AutoRest/blob/master/Documentation/swagger-extensions.md#x-ms-enum